### ISSUE TITLE
Classificaiton models dont run

### ISSUE THREAD

**Issue by knussear** [gbm-developers/gbm3#164]
Trying to run a classification model. 

Simple example per your code demo here [gbm3::printExamples](http://127.0.0.1:46785/help/library/gbm3/demo/printExamples.r)

data( iris )
kMod <- gbm( Species ~ . , data=iris , distribution = "gaussian", n.tree=1000, shrinkage=.1,
             cv.folds=5, train.fraction=.9)
results in 
"Error: Not compatible with requested type: [type=character; target=double]."

checked the Species variable, and it is a factor. Not sure how to proceed, seems like an error.

**Comment by gregridgeway** [comment]
You're asking to gbm to minimize squared error loss (gaussian), but you need a continuous outcome.No gbm3 option for factor outcomesGreg
-------- Original message --------From: knussear ***@***.***> Date: 5/1/24  9:40 AM  (GMT-05:00) To: gbm-developers/gbm3 ***@***.***> Cc: Subscribed ***@***.***> Subject: [gbm-developers/gbm3] Classificaiton models dont run (Issue #164) 
Trying to run a classification model.
Simple example per your code demo here gbm3::printExamples
data( iris )
kMod <- gbm( Species ~ . , data=iris , distribution = "gaussian", n.tree=1000, shrinkage=.1,
cv.folds=5, train.fraction=.9)
results in
"Error: Not compatible with requested type: [type=character; target=double]."
checked the Species variable, and it is a factor. Not sure how to proceed, seems like an error.

**Comment by knussear** [comment]
Yet this is literally directly out of your examples document. Is this no longer possible in gbm? I think the old one used multinomial distributions, but this is no longer an option.

**Comment by gregridgeway** [comment]
Yikes! Is it really in a gbm3 example? Where is that? I don't see it in the vignettes or the help file.
________________________________

**Comment by knussear** [comment]
Yes - Here is where I found it. In the code demos link at the top of the package help there is a link to the print examples, and within that file there is the code I gave above
